cytotable/convert.py

```diff
@@ -1314,6 +1314,8 @@ def convert(  # pylint: disable=too-many-arguments,too-many-locals
             an optional group of presets to use based on common configurations
         parsl_config: Optional[parsl.Config] (Default value = None)
             Optional Parsl configuration to use for running CytoTable operations.
+            Note: when using CytoTable multiple times in the same process the
+            first configuration loaded will be reused during subsequent runs.
```

Comment:
```suggestion
            Note: when using CytoTable multiple times in the same process,
            CytoTable will use the first provided configuration for all runs.
```

I made some edits to increase clarity, but it's possible my suggestion doesn't quite hit the mark, so feel free to reject/edit.

Reply:
Thanks - I think this is a lot more clear! I'll add this suggestion as a commit.